src/Tools/generateTemplates/templateClassPyExport.py

```diff
@@ -1125,6 +1139,21 @@ class @self.export.Namespace.replace("::","_")@Export @self.export.Name@ : publi
     return 0; 
 }
 -
+
++ if (self.export.DescriptorGetter):
+PyObject* @self.export.Name@::descriptorGetter(PyObject* self, PyObject* obj, PyObject* type)
+{
+    PyErr_SetString(PyExc_NotImplementedError, "Not yet implemented");
+    return nullptr;
+}
+-
++ if (self.export.DescriptorSetter):
+int @self.export.Name@::descriptorSetter(PyObject* self, PyObject* obj, PyObject* value)
+{
+    PyErr_SetString(PyExc_NotImplementedError, "Not yet implemented");
+    return nullptr;

```

Comment:
This generates invalid code. The return type is an int not a pointer.

Reply:
I'll fix it right now